libs/extensions/std/lang/src/text-file-interpreter-meta-inf.spec.ts

```diff
@@ -0,0 +1,53 @@
+// SPDX-FileCopyrightText: 2023 Friedrich-Alexander-Universitat Erlangen-Nurnberg
+//
+// SPDX-License-Identifier: AGPL-3.0-only
+
+import {
+  createJayveeServices,
+  useExtension,
+} from '@jvalue/jayvee-language-server';
+import {
+  TestLangExtension,
+  ValidationResult,
+  readJvTestAssetHelper,
+  validationHelper,
+} from '@jvalue/jayvee-language-server/test';
+import { AstNode } from 'langium';
+import { NodeFileSystem } from 'langium/node';
+
+import { StdLangExtension } from './extension';
+
+describe('Validation of TextFileInterpreterMetaInformation', () => {
+  let validate: (input: string) => Promise<ValidationResult<AstNode>>;
+
+  const readJvTestAsset = readJvTestAssetHelper(__dirname, '../test/assets/');
+
+  beforeAll(() => {
+    // Register std extension
+    useExtension(new StdLangExtension());
+    // Register test extension
+    useExtension(new TestLangExtension());
+    // Create language services
+    const services = createJayveeServices(NodeFileSystem).Jayvee;
+    // Create validation helper for language services
+    validate = validationHelper(services);
+  });
+
+  it('should diagnose error on invalid encoding parameter value', async () => {
```

Comment:
Does it make sense to add one with a valid encoding parameter?

Reply:
Or for all valid ones with `it.each`?

Reply:
Considering that the code that directly determines whether the encoding is valid wasn't written by us I think that would not be necessary.